js/src/lib/ajax-filter.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @package Polylang
+ */
+
+/**
+ * Adds data to all ajax requests made with jQuery.
+ *
+ * @since 3.7
+ *
+ * @param {Object} data The data to add.
+ * @returns {void}
+ */
+const ajaxFilter = ( data ) => {
+	if ( 'undefined' === typeof jQuery ) {
```

Comment:
```suggestion
	if ( 'undefined' === typeof jQuery || ! data ) {
```
Follows my previous comment.

Reply:
Well, I thought it was a good idea to check for `length` prototype property as well.
But it seems to exists only for strings or arrays, so it'd fail for object.
Do you confirm we don't want that? Object must pass?

Reply:
Oh, right, I brainfarted. I thought it was available for objects and not for arrays for a moment >_<
Fixed in [cc6b1ee](https://github.com/polylang/polylang/pull/1612/commits/cc6b1eeedc8140db3a7a5443d79c7aa59787f1b1).

Reply:
🤣 
And also, if somedays prototype property are not found, we're in deep shit regarding Javascript in general 😅

Reply:
You know that I don't understand shit about what you just said, right? 😅